mfxconfig.h.in

```diff
@@ -44,3 +44,5 @@
 #cmakedefine MFX_ENABLE_MJPEG_VIDEO_ENCODE
 #cmakedefine MFX_ENABLE_VC1_VIDEO_DECODE
 #cmakedefine MFX_ENABLE_AV1_VIDEO_DECODE
+
+#cmakedefine MFX_ENABLE_ENCTOOLS
```

Comment:
This one is not seems hanging in the air. You only expose cmake option for MFX_ENABLE_AENC, but specify MFX_ENABLE_ENCTOOLS here and don't have MFC_ENABLE_AENC at all. This confuses a lot!

Let's just stop all that and have single MFX_ENABLE_ENCTOOLS for everything?